go/vt/vtgate/executor_test.go

```diff
@@ -1818,6 +1818,46 @@ func TestGetPlanNormalized(t *testing.T) {
 	assertCacheContains(t, r, unshardedvc, normalized)
 }
 
+func TestGetPlanPriority(t *testing.T) {
+
+	testCases := []struct {
+		name             string
+		sql              string
+		expectedPriority string
+		expectedError    error
+	}{
+		{name: "empty priority", sql: "select * from music_user_map", expectedPriority: "", expectedError: nil},
+		{name: "Invalid priority", sql: "select /*vt+ PRIORITY=something */ * from music_user_map", expectedPriority: "", expectedError: sqlparser.ErrInvalidPriority},
+		{name: "Valid priority", sql: "select /*vt+ PRIORITY=33 */ * from music_user_map", expectedPriority: "33", expectedError: nil},
```

Comment:
add `{name: "empty priority", sql: "select * from music_user_map", expectedPriority: "", expectedError: nil},` after the last test case of `33`. Does it reset the priority ?

Reply:
Good catch. Re-ordered and had to modify `vcursorImpl.SetPriority()` so that it is set per query.